services/src/test/scala/cromwell/pipeline/service/GitLabProjectVersioningTest.scala

```diff
@@ -33,25 +32,48 @@ class GitLabProjectVersioningTest extends AsyncWordSpec with ScalaFutures with M
       }
       "return new active Project with 201 response" taggedAs Service in {
         val project = projectWithRepo
-        when(request(activeProject)).thenReturn(Future.successful(Response(StatusCodes.Created.intValue, "", Map())))
+        when(request(activeProject)).thenReturn(Future.successful(Response(HttpStatusCodes.Created, "", Map())))
```

Comment:
It's better to create a value for it, for example `emptyBody`  and define it as "". Even better is to add some sense to this definition and give up idea of empty strings usage.

Reply:
Fixed! I made EmptyBody - cause it is constant.